### DISCUSSION TITLE
Unable to connect to PG Schema [Jun 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2473] [Discord Threads]
> 🙋‍♂️ **lunax**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Could you help us with additional information   
#### - NocoDB project information (https://www.youtube.com/watch?v=AUSNN-RCwhE)   
#### - Working on self-hosted version? or using cloud service provider?  
#  
> 🙋‍♂️ **lunax**  
#### selfhosted:   
#### ```   
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **pg**   
#### ProjectOnRootDB: **false**   
#### RootDB: **pg**   
#### PackageVersion: **0.91.10**```  
#  
> 🙋‍♂️ **lunax**  
#### But now started to work...  
#  
> 🙋‍♂️ **lunax**  
#### I was trying for past hour and just randomly started to work.  
#  
> 👨‍🦲 **dstala**  
#### any error that you saw?  
#  
> 🙋‍♂️ **lunax**  
#### Then? No, just empty tables list.  
#  
> 🙋‍♂️ **lunax**  
#### and when I creating tables it created it in public schema.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/986625231626067968)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
